src/content/docs/en/guides/migrate-to-astro/from-docusaurus.mdx

```diff
@@ -64,10 +64,7 @@ You can pass a `--template` argument to the `create astro` command to start a ne
     </Fragment>
   </PackageManagerTabs>
 
-
-Add our MDX integration and bring your existing content files to [create MDX pages](/en/guides/markdown-content/). You can still take advantage of [file-based routing](/en/core-concepts/routing/) by copying these documents directly into `src/pages/` in Astro, the same folder you currently use. Create folders with names that correspond to your existing Docusaurus project, and you should be able to keep your existing URLs. 
-
-Docusaurus probably handled much of your site layout and metadata for you. You may wish to read about [building Astro Layouts as page wrappers for Markdown and MDX](/en/core-concepts/layouts/#markdownmdx-layouts) to see how to manage templating yourself in Astro, including your page `<head>`.
+Add our MDX integration and [bring your existing content files to Starlight](https://starlight.astro.build/getting-started/#add-content).
```

Comment:
Starlight includes the MDX integration already, so technically not required? Or should this be an OR instead of an and?

Reply:
Good call! The original Astro would have needed it added. Will correct!